modules/apps/object/object-rest-impl/src/main/java/com/liferay/object/rest/internal/manager/v1_0/DefaultObjectEntryManagerImpl.java

```diff
@@ -419,6 +423,55 @@ public ObjectEntry getObjectEntry(
 			dtoConverterContext, objectDefinition, objectEntry);
 	}
 
+	@Override
+	public Page<ObjectEntry> getObjectEntryRelatedObjectEntries(
+			DTOConverterContext dtoConverterContext, Long objectEntryId,
+			ObjectDefinition objectDefinition, String objectRelationshipName,
+			Pagination pagination)
+		throws Exception {
+
+		ObjectRelationship objectRelationship =
+			_objectRelationshipService.getObjectRelationship(
+				objectDefinition.getObjectDefinitionId(),
+				objectRelationshipName);
+
+		com.liferay.object.model.ObjectEntry objectEntry =
+			_objectEntryService.getObjectEntry(objectEntryId);
+
+		ObjectEntry[] objectEntries = new ObjectEntry[0];
```

Comment:
Is any special reason why we are using an array instead of list?

Reply:
Honestly? I don't remember 😅 I think it comes from one of Brian's reviews for the first PR. I will change it if I can because makes more sense as a list. Thanks!